libs/top_level_metrics.py

```diff
@@ -7,9 +7,15 @@
 from libs import series_utils
 
 
+# We will assume roughly 5 tracers are needed to trace a case within 48h.
+# The range we give here could be between 5-15 contact tracers per case.
```

Comment:
Do we have a citation somewhere in our code for the origin of the 5-15 range?

Reply:
Didn't see one in the javascript code :/